app/src/main/java/com/brandon/forzenbook/viewmodels/LoginViewModel.kt

```diff
@@ -0,0 +1,239 @@
+package com.brandon.forzenbook.viewmodels
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.util.Log
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.core.text.isDigitsOnly
+import androidx.core.util.PatternsCompat
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.brandon.forzenbook.usecase.CreateUserUseCase
+import com.brandon.forzenbook.usecase.GetLoginCodeUseCase
+import com.brandon.forzenbook.usecase.LoginUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import java.sql.Date
+import java.text.SimpleDateFormat
+import javax.inject.Inject
+
+@HiltViewModel
+class LoginViewModel @Inject constructor(
+    private val loginUseCase: LoginUseCase,
+    private val getLoginCodeUseCase: GetLoginCodeUseCase,
+    private val createUserUseCase: CreateUserUseCase,
+) : ViewModel() {
+
+    private var sqlDate: Date? = null
```

Comment:
Weird name for this

Reply:
renamed this to formattedDate to make it clearer, hopefully